includes/modules/pages/document_general_info/main_template_vars.php

```diff
@@ -70,6 +70,9 @@
   if ($product_info->fields['products_image'] != '' || PRODUCTS_IMAGE_NO_IMAGE_STATUS != '1') {
     $products_image = $product_info->fields['products_image'];
   }
+  if (!isset($ignore_products_image)) {
+    $ignore_products_image = 0;
+  }
```

Comment:
Not needed for the code specifically though not including it relies on that specific code test. It offers users a reasonable point where the value is expected to be set, troubleshooting aid, and is an appropriate location to set main template variables, unless that isn't the purpose of this file?

Waiting for the new, more specific filename, because apparently this is not the place to set main_template_vsriables.